### PR TITLE
Refactor business logic; Add unit tests

### PR DESCRIPTION
**Pull request by nickolashe**

### CODE REVIEW

File: openCurrents/ocuser.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
```
Comment (nickolashe) [inline openCurrents/ocuser.py]: @rajatkapoor We'll first be unit testing this class, which now contains the business logic related to getting user balances and event data.

Comment (nickolashe) [reply openCurrents/ocuser.py]: My plan is to add unit test scenarios and provide an example test tomorrow (10/14).

Comment (rajatkapoor) [reply openCurrents/ocuser.py]: Ok.Sure

**Comment by nickolashe** [comment]
merged into #525

**Pull request closed without merging**